packages/desktop/src/main/index.ts

```diff
@@ -49,7 +53,26 @@ const createMainWindow = () => {
         width: 1200,
     });
 
+    // Listen for renderer crashes and attempt a reload.
+    mainWindow.webContents.on("crashed", (event, killed) => {
+        console.error("Renderer process crashed. Killed:", killed);
+        if (reloadAttempts < MAX_RELOAD_ATTEMPTS) {
+            reloadAttempts++;
+            console.log(`Reloading renderer... attempt ${reloadAttempts}/${MAX_RELOAD_ATTEMPTS}`);
+            setTimeout(() => {
+                if (mainWindow) {
+                    mainWindow.reload();
+                }
+            }, 1000);
```

Comment:
maybe we want less than a second here?

Reply:
Hmm why wait at all?

Reply:
The idea was to account for async tasks that might not have finished/ happened in the wrong order.

Reply:
What tasks? If we have crashed can they still be running?